### PR TITLE
Feature/main nav

### PR DESCRIPTION
**Pull request by Xochi**
Adding the main nav to ui-kit starter

### CODE REVIEW

**Comment by sukhrajghuman** [comment]
Also in your personal opinion which do you think looks better our of these? I feel the side-nav lines go too long...?

## Option 1 (current)
image


## Option 2
image

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Australian Government Design System Starter Pack
 
 * [Install](#install)
 * [Contributing](#contributing)
-* [License](#license)
+* [Licence](#licence)
```
Comment (alex-page) [inline README.md]: Lets leave this spelling as is.

File: README.md [hunk 2]
```diff
@@ -43,9 +43,9 @@ To contribute to the Australian Government Design System Starter Pack:
 ----------------------------------------------------------------------------------------------------------------------------------------------------------------
 
 
-## License
+## Licence
 
-Copyright (c) Commonwealth of Australia. With the exception of the Commonwealth Coat of Arms and where otherwise noted, this work is licensed under the [MIT license](https://raw.githubusercontent.com/govau/uikit-starter/master/LICENSE).
+Copyright (c) Commonwealth of Australia. With the exception of the Commonwealth Coat of Arms and where otherwise noted, this work is licensed under the [MIT licence](https://raw.githubusercontent.com/govau/uikit-starter/master/LICENSE).
```
Comment (alex-page) [inline README.md]: Lets leave this spelling as is.

File: README.md [hunk 3]
```diff
@@ -43,9 +43,9 @@ To contribute to the Australian Government Design System Starter Pack:
 ----------------------------------------------------------------------------------------------------------------------------------------------------------------
 
 
-## License
+## Licence
```
Comment (alex-page) [inline README.md]: Lets leave this spelling as is.

**Comment by Xochi** [comment]
Good points Raj 👍 
I updated the branch with your feedback.

File: src/sass/main.scss [hunk 4]
```diff
@@ -75,3 +77,9 @@ body > .au-body {
 .au-footer .au-link-list li {
 	@include AU-space( padding-right, 1unit );
 }
+
+.au-side-nav {
```
Comment (alex-page) [inline src/sass/main.scss]: Nice work @Xochi, for this one I would just change the grid in the HTML. We can reduce the column size and use the offset class.

https://designsystem.gov.au/components/grid-12/#offset

**Comment by Xochi** [comment]
@alex-page Updated the html to use the grid.

**Comment by sukhrajghuman** [comment]
looks good to me :)

**Pull request merged**